waku/waku_store/rpc.nim

```diff
@@ -62,6 +62,7 @@ type
     ## the state of its request
     NONE = uint32(0)
     INVALID_CURSOR = uint32(1)
+    TOO_MANY_REQUESTS = uint32(429)
```

Comment:
same in this file, isn't this supposed to be deprecated?

Reply:
No, it should not definetely. :-) It is store protocol definition.